src/Invocation.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference path="references.ts" />
 
-var child_pty = require('child_pty');
+var pty = require('pty.js');
```

Comment:
Not really, similar to this https://github.com/black-screen/black-screen/blob/master/dts/AnsiParser.d.ts. Will you? :)